bin/MadGraph5_aMCatNLO/cards/production/13TeV/Wgamma_Monophoton_Ptg130_NLO_fxfx/Wgamma_Monophoton_Ptg130_NLO_run_card.dat

```diff
@@ -0,0 +1,152 @@
+#***********************************************************************
+# Tag name for the run (one word)                                      *
+#***********************************************************************
+  tag_1 =  run_tag  ! name of the run
+#***********************************************************************
+# Number of LHE events (and their normalization) and the required      *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+        100 = nevents
+  0.001 =  req_acc  ! Required accuracy (-1=auto determined from nevents
+  25    =  nevt_job ! Max number of events per job in event generation.
+                 !  (-1= no split).
+#***********************************************************************
+# Normalize the weights of LHE events such that they sum or average to *
+# the total cross section                                              *
+#***********************************************************************
+  average       =  event_norm       ! average or sum
+#***********************************************************************
+# Number of points per itegration channel (ignored for aMC@NLO runs)   *
+#***********************************************************************
+  0.01  =  req_acc_FO        ! Required accuracy (-1=ignored, and use the
+                           ! number of points and iter. below)
+# These numbers are ignored except if req_acc_FO is equal to -1
+  5000  =  npoints_FO_grid   ! number of points to setup grids
+
+  4     =  niters_FO_grid    ! number of iter. to setup grids
+  10000 =  npoints_FO        ! number of points to compute Xsec
+  6     =  niters_FO         ! number of iter. to compute Xsec
+#***********************************************************************
+# Random number seed                                                   *
+#***********************************************************************
+       4533 =  iseed
```

Comment:
change this to default setting iseed = 0

Reply:
Changed